### PR TITLE
Updated take action page

### PR DESCRIPTION
**Pull request by kristinli19**
@justineschott  and @jamesbaskerville  - Just some of the latest edits to the take action page -- adding in the title and full list of questions Dan drafted

### CODE REVIEW

File: pages/take-action.vue [hunk 1]
```diff
@@ -54,7 +54,16 @@
...
-            <v-row justify="center">
+            <v-row justify="center" align="center" class="my-2">
+                <div class="Take-Action-Title"> Take Action </div>
+            </v-row>
+            <v-row justify="center" align="center" class="mb-10">
+                <div class="Take-Action-Subheading"> See something you don't like? Use your voice to create change! </div>
+            </v-row>
+            <!-- <v-row class="my-5"> <v-spacer /> <v-col col="1">
+                <v-text-field label="Enter Your Zip Code" outlined placeholder="94110" light dense background-color="white"></v-text-field>
+            </v-col> <v-spacer /> </v-row> -->
```
Comment (jamesbaskerville) [inline pages/take-action.vue:65]: might as well just remove, we can find it from the git history later if we want to add back in
```suggestion
```

File: pages/take-action.vue [hunk 2]
```diff
@@ -124,6 +133,32 @@ export default Vue.extend({
 .v-application--wrap {
   justify-content: space-between;
 }
+
+.Take-Action-Title{
+  font-family: Avenir;
+  font-style: normal;
```
Comment (jamesbaskerville) [inline pages/take-action.vue:139]: nit: I think we already have the following as default styles:
```
font-family: Avenir;
font-style: normal;
```

So the only styling you need should be `font-weight`
```suggestion
```

File: pages/take-action.vue [hunk 3]
```diff
@@ -124,6 +133,32 @@ export default Vue.extend({
...
+
+  text-align: center;
+
+  color: #2A6465; 
+}
+
+.Take-Action-Subheading{
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: normal;
```
Comment (jamesbaskerville) [inline pages/take-action.vue:153]: same here
```suggestion
```

File: pages/take-action.vue [hunk 4]
```diff
@@ -124,6 +133,32 @@ export default Vue.extend({
...
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 800;
+  font-size: 24px;
+  line-height: 33px;
+  /* identical to box height */
+
+  text-align: center;
+
+  color: #2A6465;
```
Comment (jamesbaskerville) [inline pages/take-action.vue:147]: nit: use sass variable `$dark-turquoise` for readability
```suggestion
  color: $dark-turquoise;
```

File: pages/take-action.vue [hunk 5]
```diff
@@ -124,6 +133,32 @@ export default Vue.extend({
...
+.Take-Action-Subheading{
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 18px;
+  line-height: 25px;
+  /* identical to box height */
+
+
+  color: #000000;
```
Comment (jamesbaskerville) [inline pages/take-action.vue:159]: nit: use sass variable
```suggestion
  color: $black-1;
```

**Comment by jamesbaskerville** [comment]
In general, screenshots would be great! makes it easier to parse the code

File: pages/take-action.vue [hunk 6]
```diff
@@ -54,7 +54,16 @@
                     </v-btn>
                 </div>
             </v-row>
-            <v-row justify="center">
+            <v-row justify="center" align="center" class="my-2">
```
[the remark this answers is not in the document]
Comment (kristinli19) [reply pages/take-action.vue:57]: Using CSS spacing helpers ([documentation here](https://vuetifyjs.com/en/styles/spacing/))
 
m controls margin, y applies to both top and bottom, and 2 sets margin to 8px :)

File: pages/take-action.vue (hunk 1 above)
Comment (kristinli19) [reply pages/take-action.vue:65]: cool makes sense for keeping everything cleaner

File: components/ContactUs.vue [hunk 7]
```diff
@@ -1,32 +1,69 @@
...
-                {quest: "When and where can I attend public meetings to voice my opinion?", ans: "At XYZ and ABC public forums every 123 days."},
-                {quest: "Test test test?", ans: "Test test test!"}
-            ]
-        }
     }
 }
 </script>
 
 <style>
+@font-face {
```
[the remark this answers is not in the document]
Comment (kristinli19) [reply components/ContactUs.vue:54]: Definitely agreed

**Pull request merged**